node-repository/src/main/java/com/yahoo/vespa/hosted/provision/NodeList.java

```diff
@@ -87,6 +87,12 @@ public NodeList childrenOf(Node parent) {
                     .collect(collectingAndThen(Collectors.toList(), NodeList::new));
     }
 
+    public NodeList configServers() {
```

Comment:
There's already `public NodeList nodeType(NodeType nodeType)` in this class, the caller should use that instead.

Reply:
Fixed.